test/unit/math/opencl/multiply_test.cpp

```diff
@@ -202,64 +202,64 @@ TEST(MathMatrix, matrix_vector_tri_small) {
   stan::math::matrix_cl m0_cl(4, 1);
 
   m0 = m * v.triangularView<Eigen::Lower>();
-  m0_cl = stan::math::opencl::multiply<stan::math::TriangularViewCL::Entire,
-                                       stan::math::TriangularViewCL::Lower>(
-      m_cl, v_cl);
+  m_cl.triangular_view(stan::math::TriangularViewCL::Entire);
```

Comment:
The opencl/multiply test file is somewhat huge and I think with this change we could make some helper functions and than just call them with (size, tri_view_a, tri_view_b) to simplify this code. This might be is out of scope for this PR. I will open an issue for this if you feel this should be solved separately.

Reply:
A separate PR is probably preferable.